src/css/_acquia_lift.navbar.styles.scss

```diff
@@ -0,0 +1,397 @@
+/**
+ * @file
+ * Styles for integrating Personalize with the Navbar module.
+ */
+
+/**
+ * Personalize top-level icon.
+ */
+.navbar-bar .navbar-icon-acquia-lift:before {
+  background-image: url("../images/bebebe/rocket.svg");
+}
+
+.no-svg .navbar-bar .navbar-icon-acquia-lift:before {
+  background-image: url("../images/bebebe/rocket.png");
+}
+
+.navbar-bar .navbar-icon-acquia-lift {
+  &:active:before, &.navbar-active:before {
```

Comment:
I saw that once I got further down the review. I completely missed the name change and that it was a partial.